src/components/Subnavigation/SubnavigationItem.vue

```diff
@@ -42,11 +52,23 @@ export default {
     disabled: {
       type: Boolean,
       default: false
+    },
+    stringToMatch: {
+      type: String,
+      default: ''
     }
   },
   computed: {
     active () {
-      return this.matchQueryString ? this.$route.fullPath === this.to : this.$route?.fullPath?.includes(this.to);
+      if (this.stringToMatch) {
+        const pattern = new RegExp(`${this.stringToMatch}`, 'gi');
+        if (pattern.test(this.$route.fullPath)) {
+          return true;
+        }
+      }
+      return this.matchQueryString ?
+        this.$route.fullPath === this.to :
+        this.$route?.fullPath?.includes(this.to);
```

Comment:
I know you didn't write this but if the optional chaining operators are not needed in `this.$route.fullPath === this.to`, then we do not need them in `this.$route?.fullPath?.includes(this.to);`
```suggestion
        this.$route.fullPath.includes(this.to);
```